pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1209,7 +1209,7 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, allowEmul
 		return nil, err
 	}
 
-	if err := network.Sync(domain, oldSpec, dom, vmi, options); err != nil {
+	if err := network.Sync(domain, oldSpec, dom, vmi, options.InterfaceDomainAttachment); err != nil {
```

Comment:
Could you please elaborate why was the test of `options != nil` was removed?

Reply:
Additionally, the value was received using a method, and now it is being read directly.